src/modules/account/components/EditFolderTitle/EditFolderTitle.tsx

```diff
@@ -107,19 +110,30 @@ const EditFolderTitle: FC<EditFolderTitleProps> = ({
 		return (
 			<>
 				{buttons.filter((b) => b.before).map((b) => b.node)}
-				<Button
-					key={'edit-title'}
-					onClick={handler}
-					className={styles['c-edit-folder-title__edit']}
-					variants={['silver']}
-					name={tText(
-						'modules/account/components/edit-folder-title/edit-folder-title___map-aanpassen'
-					)}
-					icon={<Icon name={IconNamesLight.Edit} aria-hidden />}
-					aria-label={tText(
-						'modules/account/components/edit-folder-title/edit-folder-title___titel-aanpassen'
-					)}
-				/>
+				<Tooltip position="top">
```

Comment:
I'd add some extra functionality to the button component so you're able to add the tooltip if wanted. This way, way we don't neet to write the same code over and over again ;)

Reply:
yes or a wrapper: <ButtonWithTooltip>